package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "if-html",
+  "version": "1.0.0",
+  "description": "Practice in HTML/CSS fot Intellect Fox courses",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack-dev-server --config config/webpack.dev.js",
+    "build": "webpack --config config/webpack.prod.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Merildew/if-html.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Merildew/if-html/issues"
+  },
+  "homepage": "https://github.com/Merildew/if-html#readme"
+}
```

Comment:
Please add the libraries you used in dependencies or devDependencies.  Scripts do not work because libraries are not specified

Reply:
i know it doesnt work. json file was just a test and i forgot to remove it, so its my fault. project works without it, because i didnt use boilerplate, just rewrote html with pug(which is lodaded in global npm, not local). so pls imagine that this file doesnt exists, i will fix it in the next PR. ty